plugin/js/parsers/FictioneerParser.js

```diff
@@ -112,11 +112,35 @@ class FictioneerParser extends Parser {
     }
 
     removeUnwantedElementsFromContentElement(element) {
-        util.removeElements(element.querySelectorAll("iframe, .eoc-chapter-groups, .chapter-nav"));
+        util.removeElements(element.querySelectorAll("iframe, .eoc-chapter-groups, .chapter-nav, .related-stories-block"));
         super.removeUnwantedElementsFromContentElement(element);
     }
 
     getInformationEpubItemChildNodes(dom) {
         return [...dom.querySelectorAll(".story__header, .story__summary")];
     }
 }
+
+class LilyOnTheValleyParser extends FictioneerParser {
+    constructor() {
+        super();
+    }
+
+    customRawDomToContentStep(chapter, content) {
+        content.querySelectorAll("*").forEach(element => {
```

Comment:
Should not be doing this in WebToEpub.  This is "anti-copying" code.  Removing it would be a violation of Googles ToC.
Instead, we have provided a script for EpubEditor to remove this.